gremlin-dotnet/src/Gremlin.Net/Gremlin.Net.csproj

```diff
@@ -67,14 +67,19 @@ NOTE that versions suffixed with "-rc" are considered release candidates (i.e. p
     <RepositoryUrl>https://github.com/apache/tinkerpop</RepositoryUrl>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>    
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <EnablePackageValidation>true</EnablePackageValidation>
+    <PackageValidationBaselineVersion>3.5.2</PackageValidationBaselineVersion>
```

Comment:
When merging this to `master`, we'll have to see whether we already have additional breaking changes there. Then we can decide whether we want to change the baseline version there to `3.6.0` (as soon as that's released) to only check for breaking changes within the 3.6 line or keep it as is to check for breaking changes also between 3.6 and 3.5.